src/pdf-utils.js

```diff
@@ -36,3 +39,60 @@ export const encryptPDF = (path, password) =>
       resolve();
     })
   })
+
+export async function watermarkPDF(pdfFile, watermarkURL, startPage, skipPages) {
+  if(watermarkURL!=''){
+    const imageOption = {}
+    const imageBytes = await fetch(watermarkURL).then(res => res.arrayBuffer())
+    const dataBuffer = fs.readFileSync(pdfFile)
+    
+    
+    const pdfDoc = await PDFDocument.load(dataBuffer)
+    let image = undefined
+
+    if (watermarkURL.includes('.png')) {
+      image = await pdfDoc.embedPng(imageBytes);
+    } else {
+      image = await pdfDoc.embedJpg(imageBytes);
+    }
+
+    const imageHeight = 32;
+    const pages = pdfDoc.getPages()
+    const numberOfPages = pages.length;
+    const imageDms = image.size()
+    
+    for (let i = startPage; i < numberOfPages; i++) {
+      if (!skipPages.includes(i+1)){
+        const page = pages[i];
+        const { width, height } = page.getSize();
+        
+        await page.drawImage(image, {
+          x: width - (imageDms.width*(imageHeight/imageDms.height)) - imageHeight/2, 
+          y: height  - (imageHeight) - imageHeight/2,
+          height: imageHeight,
+          width: (imageDms.width*(imageHeight/imageDms.height)),
+          opacity: 0.6,
+          ...imageOption,
+        });
+      }
+    }
+    
+    fs.writeFileSync(pdfFile, await pdfDoc.save());
+  }
+}
+
+async function countPages(pdfFile) {
+  const pdfDoc = await PDFDocument.load(fs.readFileSync(pdfFile));
+  const pages = pdfDoc.getPages();
+  const numberOfPages = pages.length;
+  return numberOfPages;
+}
+
+export async function getCurrentCount(html, filename){
```

Comment:
we don't need the filename parameter here. It can be a tmp file (created in /tmp folder). And it can be deleted right after `countPages(filename);`